docs/docs/events/builtin/extra/security_bprm_check.md

```diff
@@ -0,0 +1,30 @@
+# security_bprm_check
```

Comment:
Thanks for adding this.
For it to be exposed in the docs, you need to add it to the navigation bar as well.